cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that the `urls.py` files in both the `cinema` and `user` apps have the correct namespaces defined (`namespace="cinema"` and `namespace="user"`). This is crucial for the `include` function to work properly with namespaces and ensure that URL routing functions as expected.